---
 src/Baballonia/Utils/Utils.cs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Baballonia/Utils/Utils.cs b/src/Baballonia/Utils/Utils.cs
index 6d6ca94..18a4655 100644
--- a/src/Baballonia/Utils/Utils.cs
+++ b/src/Baballonia/Utils/Utils.cs
@@ -50,7 +50,7 @@ public static class Utils

     public static readonly bool HasAdmin = OperatingSystem.IsWindows() && new WindowsPrincipal(WindowsIdentity.GetCurrent()).IsInRole(WindowsBuiltInRole.Administrator);

-    public static readonly string UserAccessibleDataDirectory = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments), "ProjectBabble");
+    public static readonly string UserAccessibleDataDirectory = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), "ProjectBabble");

     public static readonly string PersistentDataDirectory = IsSupportedDesktopOS
         ? Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), "ProjectBabble")
--
2.51.0
